### PR TITLE
Rip as much of Strawberry's CI/CD as possible for Sqlalchemy mapper.

### PR DESCRIPTION
**Pull request by mattalbr**
* Removed all the tweet stuff as it doesn't feel relevant.
* Involved switching to poetry, which is a win.
    
* Github actions are going to fail until we add the secrets. Hoping Patrick can do that since I don't think I have access.

### CODE REVIEW

File: .github/bot-action/main.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    tweet = base64.b64decode(tweet).decode("utf-8")
+
+
+mutation_input = {
+    "prNumber": int(os.environ["INPUT_PR_NUMBER"]),
+    "status": os.environ["INPUT_STATUS"],
+    "releaseCardUrl": os.environ.get("INPUT_RELEASE_CARD_URL"),
+    "tweet": tweet,
+    "releaseInfo": release_info,
+}
```
Comment (patrick91) [inline .github/bot-action/main.py:36]: uh, I think I need to update the API to support this repo too

Comment (mattalbr) [reply .github/bot-action/main.py:36]: Found the code, I see! I'll send you a PR updating that as well today. Should be easy!

File: .github/workflows/ok-to-preview.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  ok-to-preview:
+    if: ${{ github.event.label.name == 'ok-to-preview' }}
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Get changed files
+      #uses: lots0logs/gh-action-get-changed-files@2.1.4
+      uses: Mineflash07/gh-action-get-changed-files@feature/support-pr-target-event # Remove as soon as PR is merged
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (patrick91) [inline .github/workflows/ok-to-preview.yml:17]: I don't think this workflow is relevant at the moment 😊

**Review by patrick91 — APPROVED** [review]
I think I added all the tokens now, feel free to merge this!

File: .github/workflows/ok-to-preview.yml (hunk 2 above)
Comment (mattalbr) [reply .github/workflows/ok-to-preview.yml]: Removed!

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**